ledger/cow.go

```diff
@@ -153,6 +178,9 @@ func (cb *roundCowState) commitToParent() {
 	for txl, expires := range cb.mods.txleases {
 		cb.commitParent.mods.txleases[txl] = expires
 	}
+	for aidx, delta := range cb.mods.assets {
+		cb.commitParent.mods.assets[aidx] = delta
```

Comment:
If an asset is created and deleted within the same block, you might end up with a "deleted" delta for an asset that never had a "created" delta.  This might confuse accountsdb calling `deleteAssetIdxStmt`, modulo that call being altogether missing.

Reply:
I did think about this case, but it didn't seem like it would cause an issue? Regardless, I will eliminate it from the delta entirely so we don't have to worry about it.

Reply:
I was thinking that `deleteAssetIdxStmt` should never fail.

Reply:
Right, `DELETE`s that don't match any rows don't fail.

Reply:
Ah, indeed; that's a bit unfortunate (would have been a nice sanity check), but probably fine as-is.  Doesn't seem like we can screw up too badly since asset IDs are unique over time.